test/meter/meter-linear.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+		<title>d2l-meter-linear unit tests</title>
+		<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
I think these relative paths will likely need to be one level deeper as a result of the package scoping `@brightspace-ui/core`.  ex.  `../../../../mocha/mocha.js`.  You may want to refresh your local if looking at existing.